File: src/3392.count-subarrays-of-length-three-with-a-condition.3685/solution.java
class Solution {
    public int countSubarrays(int[] nums) {
        int result = 0;
        for(int i=0;i+2< nums.length;i++){
            if((nums[i]+nums[i+2])*2 == nums[i+1]){
                result++;
            }
        }
        return result;
    }
}

File: src/3392.count-subarrays-of-length-three-with-a-condition.3685/solution.ts
function countSubarrays (nums: number[]): number {
    let result = 0;
    for (let i = 0; i < nums.length - 2; i++) {
        if (nums[i + 1] === (nums[i] + nums[i + 2]) * 2) {
            result++;
        }
    }
    return result;
}
